### ISSUE TITLE
Still needed for Civicrm 4.5/4.6

### ISSUE THREAD

**Issue by magnolia61** [EFForg/org.eff.optimizedautocomplete#3]
Is this extension still needed for civicrm 4.5/4.6 or have these optimisations found their way into core?

**Comment by joemcl** [comment]
Yes a good question. I don't know. Posting the question to https://chat.civicrm.org/civicrm/channels/extensions
